modules/oe_webtools_social_share/src/Plugin/Block/SocialShare.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_webtools_social_share\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+
+/**
+ * Provides a 'Social Share' Block.
+ *
+ * @Block(
+ *   id = "social_share",
+ *   admin_label = @Translation("Social Share"),
+ *   category = @Translation("Webtools"),
+ * )
+ */
+class SocialShare extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $social_share_json = [
+      'service' => 'share',
+      'popup' => FALSE,
+      'selection' => TRUE,
+      'to' => [
+        'more',
+        'twitter',
+        'facebook',
+        'linkedin',
+        'e-mail',
+      ],
+      'stats' => TRUE,
+    ];
+    return [
```

Comment:
The only reason why this widget is showing up when placing the block is because the other webtools submodules are enabled. Each widget needs to also attach the smart loaders library. Check for example `oe_webtools_laco_widget_page_attachments()` and attach it also here.